docs/integrations/spark.md

```diff
@@ -191,157 +185,128 @@ df.write.partitionBy("example-column").parquet(s"s3a://${repo}/${branch}/output-
-
-```scala
-spark.sparkContext.hadoopConfiguration.set("fs.s3a.access.key", "AKIAIOSFODNN7EXAMPLE")
-spark.sparkContext.hadoopConfiguration.set("fs.s3a.secret.key", "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY")
-spark.sparkContext.hadoopConfiguration.set("fs.s3a.endpoint", "https://s3.eu-central-1.amazonaws.com")
-spark.sparkContext.hadoopConfiguration.set("fs.lakefs.impl", "io.lakefs.LakeFSFileSystem")
-spark.sparkContext.hadoopConfiguration.set("fs.lakefs.access.key", "AKIAlakefs12345EXAMPLE")
-spark.sparkContext.hadoopConfiguration.set("fs.lakefs.secret.key", "abc/lakefs/1234567bPxRfiCYEXAMPLEKEY")
-spark.sparkContext.hadoopConfiguration.set("fs.lakefs.endpoint", "https://lakefs.example.com/api/v1")
-```
-  </div>
-  <div markdown="1" id="lakefs-config-tabs-xml">
-
-Make sure that you load the lakeFS FileSystem into Spark by running it with `--packages` or `--jars`,
-and then add these into a configuration file, e.g., `$SPARK_HOME/conf/hdfs-site.xml`:
-
-```xml
-<?xml version="1.0"?>
-<configuration>
-    <property>
-        <name>fs.s3a.access.key</name>
-        <value>AKIAIOSFODNN7EXAMPLE</value>
-    </property>
-    <property>
-            <name>fs.s3a.secret.key</name>
-            <value>wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY</value>
-    </property>
-    <property>
-        <name>fs.s3a.endpoint</name>
-        <value>https://s3.eu-central-1.amazonaws.com</value>
-    </property>
-    <property>
-        <name>fs.lakefs.impl</name>
-        <value>io.lakefs.LakeFSFileSystem</value>
-    </property>
-    <property>
-        <name>fs.lakefs.access.key</name>
-        <value>AKIAlakefs12345EXAMPLE</value>
-    </property>
-    <property>
-        <name>fs.lakefs.secret.key</name>
-        <value>abc/lakefs/1234567bPxRfiCYEXAMPLEKEY</value>
-    </property>
-    <property>
-        <name>fs.lakefs.endpoint</name>
-        <value>https://lakefs.example.com/api/v1</value>
-    </property>
-</configuration>
-```
-  </div>
-</div>
-
-#### Per-bucket and per-repo configuration
-
-As demonstrated above, S3 allows for per-bucket configuration. You can use this if:
-
-1. You need to use S3A directly to access data in an S3 outside of lakeFS, _and_
-1. different credentials are required to access data inside that bucket.
-
-Refer to the Hadoop AWS guide [Configuring different S3 buckets with Per-Bucket
-Configuration](https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html#Configuring_different_S3_buckets_with_Per-Bucket_Configuration).
-
-There's no need for per-repo configurations in lakeFS when all repositories are on the same
-lakeFS server. If you need to access repositories located on *multiple* lakeFS servers,
-configure multiple prefixes. For instance, you might configure both `fs.lakefs.impl` and
-`fs.lakefs2.impl` to be `io.lakefs.LakeFSFileSystem`, place separate endpoints and credentials
-under `fs.lakefs.*` and `fs.lakefs2.*`, and access the two servers using `lakefs://...` and
-`lakefs2://...` URLs.
```

Comment:
I agree that the per-bucket config text is confusing, but thin that we should lose this part 

> If you need to access repositories located on *multiple* lakeFS servers,
configure multiple prefixes. For instance, you might configure both `fs.lakefs.impl` and
`fs.lakefs2.impl` to be `io.lakefs.LakeFSFileSystem`, place separate endpoints and credentials
under `fs.lakefs.*` and `fs.lakefs2.*`, and access the two servers using `lakefs://...` and
`lakefs2://...` URLs.

Maybe we can call it something like lakeFS filesystem configuration in a multi-instance environment. WDYT?

Reply:
I think this belongs in other community channel, rather than in the docs.